src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/ScriptActionsOnRunningCluster.cs

```diff
@@ -122,7 +122,7 @@ public void TestScriptActionsOnRunningCluster()
                     var result = client.Clusters.ExecuteScriptActions(resourceGroup, dnsName, executeScriptActionParams);
                     Assert.Equal(result.StatusCode, HttpStatusCode.OK);
                     Assert.Equal(result.State, AsyncOperationState.Failed);
-                    Assert.Equal(result.ErrorInfo.Message, "ScriptExecutionFailed");
```

Comment:
@lyerrb this is OK, if you file an issue for updating the test in github and assign it appropriately,  Please include the issue # here